ui/js/controllers/logviewer.js

```diff
@@ -1,3 +1,5 @@
+import createHashHistory from 'history/createHashHistory';
```

Comment:
I guess at some point in the future we'll switch to `history/createBrowserHistory` and make the breaking URL change?

Reply:
Yeah, I think when we transition to a React app, then we'll make that change.  My *hope* is that we can figure out a way to accept the hash urls, but convert them to non-hash in-place, as we are with the change to the filter params.  Even if it means we have to reload the page once to get there, it'd be nicer.  Maybe pop up a notification to the user that the url format was changed for them, so use this new style going forward.  I haven't experimented with that yet, but that would be nice.  :)